pytest/tests/mocknet/sharded_bm.py

```diff
@@ -354,6 +357,31 @@ def start_nodes(args, enable_tx_generator=False):
     start_nodes_cmd(CommandContext(start_nodes_cmd_args))
 
 
+def apply_network_config(args):
+    """Apply network configuration optimizations."""
+    logger.info("Applying network configuration optimizations")
+    
+    run_cmd_args = copy.deepcopy(args)
+
+    # This command does the following:
+    # - Set overrides for some configs
+    # - Update default interface to use larger initial congestion window
+    run_cmd_args.cmd = "\
+        sudo sysctl -w net.core.rmem_max=134217728 && \
+        sudo sysctl -w net.core.wmem_max=134217728 && \
+        sudo sysctl -w net.ipv4.tcp_rmem='10240 87380 134217728' && \
+        sudo sysctl -w net.ipv4.tcp_wmem='4096 87380 134217728' && \
+        sudo sysctl -w net.core.netdev_max_backlog=30000 && \
+        sudo sysctl -w net.ipv4.tcp_no_metrics_save=1 && \
+        gw=$(ip route show default | head -1 | awk '{{print $3}}') && \
+        dev=$(ip route show default | head -1 | awk '{{print $5}}') && \
+        sudo ip route flush table main exact 0.0.0.0/0 2>/dev/null || true && \
```

Comment:
You don't need to remove any routes, adding a route with a low metric should be sufficient for it to be picked first.